library/cwm/src/lib/cwm/multi_status_selector.rb

```diff
@@ -397,12 +395,15 @@ def auto_select!
 
       # Returns richtext representation for the item
       #
-      # Basically, an string containing two <a> or <span> tags, depending on the #enabled? method.
+      # Basically a string containing a table row with two <a> or <span> tags,
+      # depending on the #enabled? method.
       # One for the check box input and another for the label.
       #
       # @return [String] the item richtext representation
       def to_richtext
-        "#{checkbox_input} #{checkbox_label}"
+        # add extra space in graphical mode
+        space = Yast::UI.TextMode ? "" : " "
+        "<tr><td>#{checkbox_input}#{space}</td><td> #{checkbox_label}</td></tr>"
```

Comment:
Another way to do it could be:

```suggestion
        gap = Yast::UI.TextMode ? " " : "  "
        "<tr><td>#{checkbox_input}#{gap}</td><td>#{checkbox_label}</td></tr>"
```

If there is a problem having two spaces in a row, we can use

```
gap = Yast::UI.TextMode ? "&nbsp;" : "&nbsp;&nbsp;"

```

Reply:
It turned out that Qt displays `&nbsp;` a bit wider than a normal space so it is enough to have just one `&nbsp;` in both Qt and ncurses.